magic/src/main/scala/com/fortysevendeg/scalacheck/magic/Magic.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * scalacheck-datetime
+ * Copyright (C) 2016-2017 47 Degrees, LLC. <http://www.47deg.com>
+ */
+
+package com.fortysevendeg.scalacheck.magic
+
+import org.scalacheck._
+import org.scalacheck.Gen._
+import org.scalacheck.Arbitrary.arbitrary
+
+object Magic {
+
+  private[magic] val strings = List("This", "is", "a", "temporary", "list", "of", "strings")
+
+  val magicStrings: Gen[String] = oneOf(strings)
+
+  implicit val enhancedStringArbitrary: Arbitrary[String] = Arbitrary {
```

Comment:
I'll have a think about this. In my head the reasoning is you want arbitrary `String`s, and just have the added bonus benefit of checking some magic ones too. I see having an import to override the scalacheck default is essentially the same as tagging.